firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -93,8 +93,7 @@ class FakespotTests: BaseTestCase {
         if #available(iOS 17, *) {
             // Open a product detail page and check the address bar
             let searchField = app.webViews["contentView"].webViews.textFields["Search for anything"]
-            searchField.waitAndTap()
-            searchField.typeText("Shoe")
+            searchField.tapAndTypeText("Shoe")
```

Comment:
I am sure you have run the tests and test that out, but usually the CI environment may be slower and we need the element to be present, then tap and have it ready before typing... with this way, are we sure the element is ready to type? 
Thanks!